Kate advantages

Kaka8548 opened the pull request:
На цей раз точно все перевірила, є деякі неточності, але я вважаю, що це помилка дизайнера: 
 - в таблет та десктоп-версіях "кг" написане маленькими літерами, хоча в мобайл аперкейс
 - є неточності у відступах, але вони тупо скрісь різні 🙃 тому брала середнє значення 

Ще чомусь в мене текст Монсерат відображається не так, як на макеті (підзаголовки саме, там де кілограми і тд). Перевір, будь ласка, чи в тебе також так?

Kaka8548 committed:
Додано розмітку секції "Advantages" для мобільної версії

Kaka8548 committed:
Надано стилів секції "Advantages" для мобільної версії

Kaka8548 committed:
Додано стилів секції "Advantages" для мобільної версії

Kaka8548 committed:
Додано стилів для таблет-версії секції "Advantages"

Kaka8548 committed:
Додано стилів секції "Advantages" для десктоп версії

Kaka8548 committed:
Виправлено розміри шрифтів для таблет- та декстоп-версій

Vlad-Moskalenko reviewed (commented):
https://www.loom.com/share/4fc9408fc1d4454a91024d25b3755259

Kaka8548 committed:
Merge branch 'main' into kate-advantages

Kaka8548 committed:
Виправлено помилки

 - спільні властивості винесено в батьківський селектор
 - прибрано зайві класи
 - змінено тег підзаголовків
 - змінена відстань елементів

Kaka8548 commented:
- виправила помилки 
 - код у валідаторі перевірила
